### PR TITLE
Update test-data and e2e

### LINKED ISSUE #283: Do a E2E selecting two members of a trio and then downloading pre-signed URL manifest zip
Testing
- support for multi member VCFs
- presigning (the actual links do not have to be downloadable! just that they are vaguelly well formed - we'll have fake ACCESS keys in the test suite config anyhow)
- basic release manifest

### PR DESCRIPTION
**Pull request by williamputraintan**
- Refactor insert-test-data structure (allowing to support more scenario)
- Add test e2e test to check on manifest downloaded

### CODE REVIEW

File: e2e/tests/manager/access-release.spec.ts [hunk 1]
```diff
@@ -62,6 +70,12 @@ test("Download metadata manifest ZIP file", async ({ page }) => {
 
   // Reading the data from the stream
   readableStream?.on("end", () => {
+    const splitLines = data.split("\n");
+
+    // Test if TSV is downloaded and contain S3 URI
     expect(data).toMatch(/s3:\/\/umccr-10f-data-dev\//i);
+
+    // Test if "ELROY" data exist, as only one of the trio is selected in this release
+    expect(splitLines[1]).toMatch(/JETSONS\tELROY/);
```
Comment (andrewpatto) [inline e2e/tests/manager/access-release.spec.ts]: are the manifest lines guaranteed to be sorted (the [1]??)
also - this doesn't prove the others in the trio _aren't_ in the release - only that ELROY *is* in the release..
(where you'd probably want to assert both)

File: application/frontend/src/pages/login-dev-page.tsx [hunk 2]
```diff
@@ -13,18 +13,18 @@ export const LoginDevPage: React.FC = () => {
       </p>
       <pre>{JSON.stringify(envRelay, null, 2)}</pre>
       <form action="/auth/login-bypass-1" method="POST" id="loginBypass1Form">
-        <button className="btn-warning" type="submit">
-          Log in (test user 1 - Administrator)
+        <button className="btn-danger" type="submit">
+          Log in (test user 1 - SuperAdmin)
```
Comment (andrewpatto) [inline application/frontend/src/pages/login-dev-page.tsx:17]: Nice! They were in the wrong order!

File: application/frontend/src/pages/login-dev-page.tsx [hunk 3]
```diff
@@ -13,18 +13,18 @@ export const LoginDevPage: React.FC = () => {
...
         <button className="btn-warning" type="submit">
-          Log in (test user 2 - Manager)
+          Log in (test user 2 - Administrator)
         </button>
       </form>
       <form action="/auth/login-bypass-3" method="POST" id="loginBypass3Form">
-        <button className="btn-danger" type="submit">
-          Log in (test user 3 - SuperAdmin)
+        <button className="btn-warning" type="submit">
+          Log in (test user 3 - Manager)
```
Comment (andrewpatto) [inline application/frontend/src/pages/login-dev-page.tsx:27]: Should we add a Login 4 for "Member"?

Comment (williamputraintan) [reply application/frontend/src/pages/login-dev-page.tsx:27]: Yeah, will add user 4!

**Review by andrewpatto — APPROVED** [review]

File: application/backend/src/test-data/mock-json.json
Comment (williamputraintan) [inline application/backend/src/test-data/mock-json.json:1]: @mmalenic Double checking on this, it is no longer in use right?

Comment (mmalenic) [reply application/backend/src/test-data/mock-json.json:1]: That's correct, it's no longer used.

I was using this to mock large audit event details before, but I think we aren't focusing on that anymore.

**Review by mmalenic — APPROVED** [review]

**Review by DoxasticFox — APPROVED** [review]

**Pull request merged**